src/MyPlot/subcommand/FillSubcommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+declare(strict_types=1);
+namespace MyPlot\subcommand;
+
+use MyPlot\forms\MyPlotForm;
+use MyPlot\forms\subforms\FillForm;
+use MyPlot\Plot;
+use pocketmine\block\Block;
+use pocketmine\command\CommandSender;
+use pocketmine\item\Item;
+use pocketmine\Player;
+use pocketmine\utils\TextFormat;
+
+class FillSubcommand extends SubCommand
+{
+	/**
+	 * @param CommandSender $sender
+	 *
+	 * @return bool
+	 */
+	public function canUse(CommandSender $sender) : bool {
+		return ($sender instanceof Player) and $sender->hasPermission("myplot.command.fill");
+	}
+
+	/**
+	 * @param Player $sender
+	 * @param string[] $args
+	 *
+	 * @return bool
+	 */
+	public function execute(CommandSender $sender, array $args) : bool {
+        if(empty($args)) {
```

Comment:
formatting

Reply:
What's the problem here?

Reply:
spaces and its indented more than the lines below